content/SCALE/SCALEUIReference/TopToolbar/Alerts/AlertSettingsScreen.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Alert Settings Screens
+title: Alert Settings Screen
```

Comment:
Add to the front matter:

description: "type some short sentence description for the article that shows up in the _index.md file after adding the children shortcode.